### ISSUE TITLE
Dynamixel xm430 does not move when button is pressed on OpenCR

### ISSUE THREAD

**Issue by agirgen** [ROBOTIS-GIT/turtlebot3#375]
ISSUE TEMPLATE ver. 0.2.0

1. Which TurtleBot3 you have?

    - [ ] Burger
    - [ ] Waffle
    - [x] Waffle Pi

2. Which SBC(Single Board Computer) is working on TurtleBot3?

    - [ ] Raspberry Pi 3
    - [ ] Intel Joule 570x
    - [X] Nvidia Jetson TX2

3. Which OS you installed in SBC?

    - [ ] Ubuntu MATE 16.04 or later
    - [ ] Raspbian
    - [X] Ubuntu 16.04 LTS for Jetson TX2

4. Which OS you installed in Remote PC?

    - [X] Ubuntu 16.04 LTS (Xenial Xerus)
    - [ ] Ubuntu 18.04 LTS (Bionic Beaver)
    - [ ] Linux Mint 18.x
    - [ ] etc (PLEASE, WRITE DOWN YOUR OS)

5. Write down software version and firmware version

    - Software version: [x.x.x]
    - Firmware version: [x.x.x]
 
6. Write down the commands you used in order

    - HERE
 
7. Copy and Paste your error message on terminal

    - HERE
  
8. Please, describe detailedly what difficulty you are in 

    - I successfully programmed both Dynamixel XM430s using the Turtlebot3_Setup_Motor utility. I then successfully flashed the Turtlebot3_Core firmware to the OpenCR Board.  I then plugged the servos into TTL port 1 and 2, plugged a 11.1 volt LiPO battery and powered on the board with the switch.  When I press Push SW1 or Push SW2 to turn or move the Turtlebot3, the servos do nothing.  In troubleshooting the problem, I loaded the OpenCR Board with USB2Dynamixel and am able to drive the motors, change settings, etc. using the Dynamixel Workbench Utility, thus ruling out there is a hardware problem with the OpenCR board or servos.

Are there additional configuration steps I need to perform with the Turtlebot3_Core source code?

Thank you for any insight you can give me on this issue!!

**Comment by kijongGil** [comment]
Hi, @agirgen  :)
Could you tell me what did you programmed DXL using the turtlebot3_setup_motor?
Did you change the DXL's ID or baudrate?

**Comment by agirgen** [comment]
Hello @kijongGil ,

Thank you for the quick reply!  Using turtlebot3_setup_motor, the IDs of the servos were changed to 1 and 2.  I do not recall if the utility changed the baud rate of the motors.  I believe it was either set to 57600 or 1000000.  After running the program, I was able to drive the motors using option 3 and 4 "test left/right motor".

Thank You

**Comment by kijongGil** [comment]
Could you check your SW1 and SW2 is broken?
You can check it easily. Go to File -> Examples -> OpenCR -> 1.Basics -> c_Digital_Read_Serial in Arduino IDE and upload it.
And then open the Serial monitor. When you push SW1 and SW2, you can see that push_state is change.

**Comment by agirgen** [comment]
Good idea, I hadn't thought of trying that.  I will attempt this when I get home from work and report my findings back to you.

**Comment by agirgen** [comment]
Ok, last night I did what you recommended and loaded the c_Digital_Read_Serial sketch and found the buttons do function as they should.

**Comment by kijongGil** [comment]
Did you check again after uploading turtlebot3_core?
If your turtlebot3 don't move, you have to check your DXLs.
I recommend using [R+ manager](http://emanual.robotis.com/docs/en/software/rplus2/manager/)

**Comment by agirgen** [comment]
I have checked the DXLs using both R+ Manager and the configuration sketch designed to program and test the DXLs for use with the Turtlebot3 (the name escapes me ATM).  In both cases, I am able to set the addresses of the DXLs to 1 and 2 and am able to get movement out of both servos.  Only the turtlebot3_core does not work for me.

**Comment by agirgen** [comment]
I followed the instructions here: http://emanual.robotis.com/docs/en/platform/turtlebot3/opencr_setup/     I was not able to get the Shell Script method to work however the Arduino IDE method worked fine.  I got the "jump_to_fw" message with no errors.

**Comment by kijongGil** [comment]
Could you tell me why you are not able to get the shell script?

**Comment by agirgen** [comment]
The shell script would not load because I was typing a command wrong :-/.  After realizing my mistake, I got the script to work.  This, however, did not fix my problem.  The DXLs would not react to me pressing the buttons.  On a whim, I loaded usb_to_dxl script and changed the baud rate of the DXLs from 1Mbps to 57.6 baud int R+.  I then reloaded the turtlebot_core script and everything works as it should regardless if I use the Arduino IDE or the script.  Thank you for your help!
